apps/meteor/app/cloud/server/functions/supportedVersionsToken/supportedVersionsToken.ts

```diff
@@ -134,6 +135,31 @@ const getSupportedVersionsToken = async () => {
 		(response.success && response.result) || undefined,
 	);
 
+	SystemLogger.debug({
+		msg: 'Supported versions',
+		supportedVersionsFromBuild: JSON.stringify(supportedVersionsFromBuild.timestamp),
+		versionsFromLicense: JSON.stringify(versionsFromLicense?.supportedVersions?.timestamp),
+		response: JSON.stringify(response.success && response.result?.timestamp),
+	});
+
+	switch (supportedVersions) {
+		case supportedVersionsFromBuild:
+			SystemLogger.info({
+				msg: 'Using supported versions from build',
+			});
+			break;
+		case versionsFromLicense?.supportedVersions:
+			SystemLogger.info({
+				msg: 'Using supported versions from license',
+			});
+			break;
+		case response.success && response.result:
+			SystemLogger.info({
+				msg: 'Using supported versions from cloud',
+			});
+			break;
+	}
```

Comment:
I guess this is not accurate logs? I mean, `supportedVersionsFromBuild` will always be defined, so it will probably always show `Using supported versions from build` ?

shouldn't this log be based on the return of `supportedVersionsChooseLatest`?